{
    "name": "exceptions_2",
    "description": "This is the test case to check if the static analysis tools work with exception handling.",
    "n_files": 3,
    "lines_of_code": 17,
    "n_classes": 0,
    "repo_url": ""
}